ieee754r/Versions

```diff
@@ -264,6 +264,12 @@ libdfp {
     quantumd128;
   }
   LIBDFP_1.0.15 {
+    nextdownd32;
+    nextdownd64;
+    nextdownd128;
+    nextupd32;
```

Comment:
check-abi-libdfp fails due to the ordering of these. Please update and double check.